.github/workflows/call-update-data-r.yml

```diff
@@ -0,0 +1,17 @@
+# Update data files in /data whenever changes are made to files in /data-raw.
+# This workflow automatically commits changes directly to the working branch.
```

Comment:
This is brilliant! I do not understand though why we are calling another file within this repository rather than just using the other file?